packages/example-app-base/src/components/ColorPalette/ColorPalette.styles.ts

```diff
@@ -1,7 +1,26 @@
-import { IRawStyle, IStyleFunction, FontWeights, getFocusStyle } from 'office-ui-fabric-react';
+import { IRawStyle, IStyleFunction, FontWeights, getFocusStyle, getGlobalClassNames } from 'office-ui-fabric-react';
 import { NeutralColors, MotionDurations, MotionTimings } from '@uifabric/fluent-theme';
 import { IColorPaletteStyleProps, IColorPaletteStyles } from './ColorPalette.types';
 
+const globalClassNames: { [key in keyof IColorPaletteStyles]: string } = {
```

Comment:
Do we really need global class names for this, since it's already customizable via the styles prop (and only used in limited places)? Unless there's a specific need for class names, I'd prefer to leave them out since we're generally trying to move away from that approach.

Reply:
I was trying to make it consistent with other css-in-js components, but if it's not needed, I can remove this.